### ISSUE TITLE
Archive uses unknown compression method

### ISSUE THREAD

**Issue by ryanfortner** [wimpysworld/deb-get#29]
When downloading and attempting to install the [latest deb](https://github.com/wimpysworld/deb-get/releases/download/0.2.2/deb-get_0.2.2-1_all.deb), I get this error.

`dpkg-deb: error: archive 'deb-get_0.2.2_all.deb' uses unknown compression for member 'control.tar.zst', giving up`

It turns out zstd compression in dpkg is *not* supported in debian bullseye stable, which is what I'm trying to run this on. Is there any way it can be packaged on a slightly older version of ubuntu so that it works on a wider range of systems?

**Comment by ryanfortner** [comment]
The latest release deb still has the same issue. 
```
dpkg-deb: error: archive 'deb-get_0.2.6-1_all.deb' uses unknown compression for member 'control.tar.zst', giving up
```

**Comment by eMPee584** [comment]
The command added in 2b32870 is commented out, so ineffective.
With docker installed, package can be repacked with

```
docker run --rm --volume=$(pwd):/tmp ubuntu dpkg-deb --raw-extract /tmp/[DEBNAME].deb /tmp/[DEBNAME]
mv [DEBNAME].deb [DEBNAME].zstd.deb
dpkg-deb --build [DEBNAME]
```
but that of course just gives "[!] ERROR! Debian is not supported.", which is a really strange feat given the name of the tool. Guess this will be fixed with #162 ..
